pkg/gitparse/gitparse.go

```diff
@@ -210,8 +210,8 @@ type Option func(*Parser)
 func NewParser(options ...Option) *Parser {
 	parser := &Parser{
 		dateFormat:    defaultDateFormat,
-		maxDiffSize:   defaultMaxDiffSize,
-		maxCommitSize: defaultMaxCommitSize,
+		maxDiffSize:   int(defaultMaxDiffSize),
+		maxCommitSize: int(defaultMaxCommitSize),
```

Comment:
why narrow casting here ? Gopher won't throw error but might loss data silently.

Reply:
Ah, yep. Good call, this will actually break on 32-bit machines. I'll create a new PR to address this. Thanks for keeping me honest.

Reply:
I should have catch it before approving the PR. Sorry about that.

Reply:
My eyes just glossed over the fact that I was undoing the earlier type change 🤣 
follow-up [PR](https://github.com/trufflesecurity/trufflehog/pull/3793)